### PR TITLE
docs: add tealium readme

### PR DESCRIPTION
**Pull request by colincclark**

### CODE REVIEW

File: packages/tealium/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```
+import tealiumReporter, {
+  TealiumSendScheduler
+} from "@times-components/tealium";
+
+const trackingOptions = {
+  enabled: true,
+  account: "my-account-name",
+  profile: "my-profile-name",
+  env: "dev",
```
Comment (craigbilner) [inline packages/tealium/README.md]: `my-env-name`?

File: packages/tealium/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  enabled: true // default is true
```
Comment (craigbilner) [inline packages/tealium/README.md]: double `enabled`

File: packages/tealium/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+};
+
+const tealiumSendScheduler = new TealiumSendScheduler(
+  trackingOptions,
+  global.window,
+  global.document
```
Comment (craigbilner) [inline packages/tealium/README.md]: minor: but `document` can probably be derived from `window`, separate PR :smirk:

File: packages/tealium/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+> Tag Management is a system for managing the configuration, testing, and
+> release of third-party vendor tags to a digital property. This approach
+> replaces the hard-coded tags on your site with a single container tag. Once
+> this tag is in place you have full control over adding, removing, and updating
+> the tags on your site directly from the iQ Tag Management interface.
+
+You can find a good overview of tag management and how it works
+[on the Tealium website](https://community.tealiumiq.com/t5/iQ-Tag-Management/Tag-Management-Concepts/ta-p/15883).
+
+Nb: This is for web only. Mobile tracking is done natively.
```
Comment (craigbilner) [inline packages/tealium/README.md]: perhaps some more around how tracking is our generic way of bubbling up events and we just happen to use Tealium in order to pump them to the backend

Comment (colincclark) [reply packages/tealium/README.md]: done

File: packages/tealium/README.md (hunk 3 above)
Comment (colincclark) [reply packages/tealium/README.md]: added a "Future" section with this suggestion

File: packages/tealium/README.md (hunk 2 above)
Comment (colincclark) [reply packages/tealium/README.md]: done

File: packages/tealium/README.md (hunk 1 above)
Comment (colincclark) [reply packages/tealium/README.md]: done

**Pull request merged**